doc/source/examples/multiphysics/stefan_problem/stefan_problem.rst

```diff
@@ -0,0 +1,184 @@
+
+----------------------------------
+Features
+----------------------------------
+- Solver: ``gls_navier_stokes_2d`` 
+- Heat transfer pĥysics
+- Unsteady problem handled by a BF1 time-stepping scheme
+- Phase change specific heat model
+
+
+------------------------
+Location of the example
+------------------------
+``examples/multiphysics/stefan_problem/stefan.prm``
+
+
+Description of the case
+-------------------------
+
+The Stefan problem describes the melting or the solidification of a pure substance by conduction in a 1D semi-infinite domain. This classical problem, extensively described in the literature [1], is often used as the core test case to establish the accuracy and the robustness of numerical models for phase change. Although it is established for a semi-infinite domain, it can be solved on a finite domain provided that it is sufficiently long. This problem is illustrated in the following figure:
+
+.. image:: images/stefan_problem_illustration.png
+    :alt: problem_illustration
+    :align: center
+
+
+
+The analytical solution for the temperature in the liquid is given by:
+
+.. math::
+
+    T(x,t) = T_W + (T_S-T_W) \frac{erf\left(\frac{x}{2\sqrt{\alpha t}} \right)}{\beta}
+
+ 
+with :math:`\alpha = \frac{k}{\rho C_p}` the thermal diffusivity, :math:`k` the thermal conductivity, :math:`\rho` the density,  :math:`T_W` is the wall temperature and :math:`T_S` is the melting (or solidification) point of a pure substance. The parameter :math:`\beta` is obtained by solving the non-linear algebraic equation:
+
+.. math::
+    \beta \exp^{\beta^2} erf \beta = \frac{St}{\sqrt{\pi}}
+
+with :math:`St` the Stefan number which is defined as :
+
+.. math::
+    St = \frac{C_p \left ( T_{W}-T_{S}\right)}{h_l}
+
+where :math:`C_p` is the specific heat (which is the same for the liquid and the solid phase) and :math:`h_l` is the latent heat. 
+
+In this example, this equation is solved using the SciPy package and Python. The displacement of the solid–liquid (the dashed lined in Fig. 1) is given by:
+
+.. math::
+
+  \delta (t) = 2 \beta \sqrt{\alpha t}
+
+
+the diffusivity coefficient in the liquid phase and :math:`\delta (t)` the melting front displacement. In this example, we will consider the case where :math:`\alpha=1`, :math:`h_l=100`, :math:`T_W=1` and :math:`T_S=0`. This corresponds to :math:`St=100`. However, the model within Lethe is sufficient robust to manage any values of the Stefan number.
+
+Although simple, this problem can be challenging to solve numerically because of the sharp impact of the phase change on the specific heat within the solidification interval. Even if this problem is inherently a 1D problem, we analyse it in 2D by generating structured quadrilateral meshes on a [0, 0]X[1, 0.1] domain. The number of nodes in the y direction is ketp at 2 (one cell), but it is adjusted to 101 in the x direction which is the direction in which the heat transfer occurs and in which the interface displaces. 
```

Comment:
```suggestion
Although simple, this problem can be challenging to solve numerically because of the sharp impact of the phase change on the specific heat within the solidification interval. Even if this problem is inherently a 1D problem, we analyse it in 2D by generating structured quadrilateral meshes on a [0, 0]X[1, 0.1] domain. The number of nodes in the y direction is kept at 2 (one cell), but it is adjusted to 101 in the x direction which is the direction in which the heat transfer occurs and in which the interface displaces. 
```